modules/tournament/src/main/TournamentApi.scala

```diff
@@ -201,15 +201,17 @@ final class TournamentApi(
     else if (tour.isCreated) destroy(tour)
     else funit
 
+  private def tourUrl(tourId: String): String = s"https://lichess.org/tournament/$tourId"
+
   private def awardTrophies(tour: Tournament): Funit = {
     import lila.user.TrophyKind._
     tour.schedule.??(_.freq == Schedule.Freq.Marathon) ?? {
       playerRepo.bestByTourWithRank(tour.id, 100).flatMap {
         _.map {
-          case rp if rp.rank == 1  => trophyApi.award(rp.player.userId, marathonWinner)
-          case rp if rp.rank <= 10 => trophyApi.award(rp.player.userId, marathonTopTen)
-          case rp if rp.rank <= 50 => trophyApi.award(rp.player.userId, marathonTopFifty)
-          case rp                  => trophyApi.award(rp.player.userId, marathonTopHundred)
+          case rp if rp.rank == 1  => trophyApi.award(tourUrl(tour.id), rp.player.userId, marathonWinner)
+          case rp if rp.rank <= 10 => trophyApi.award(tourUrl(tour.id), rp.player.userId, marathonTopTen)
+          case rp if rp.rank <= 50 => trophyApi.award(tourUrl(tour.id), rp.player.userId, marathonTopFifty)
+          case rp                  => trophyApi.award(tourUrl(tour.id), rp.player.userId, marathonTopHundred)
```

Comment:
I wasn't sure how to test this code locally. if it's blocking that I test it I can probably modify schedule.freq in `bin/mongodb/wesley02.js`  and run that locally.

Reply:
that's ok leave it